src/project/git/GitCommandGitProject.ts

```diff
@@ -195,6 +195,14 @@ export class GitCommandGitProject extends NodeFsLocalProject implements GitProje
         return this.runCommandInCurrentWorkingDirectory(`git checkout ${sha} --`);
     }
 
+    /**
+     * Revert all changes since last commit
+     * @return {any}
+     */
+    public async revert(): Promise<CommandResult<this>> {
```

Comment:
???

Reply:
Not sure what I did here but I ended up collection a bunch of things in this PR that were in master already.